src/sentry/api/endpoints/organization_config_repositories.py

```diff
@@ -13,11 +14,12 @@ def get(self, request, organization):
         providers = []
         for provider_id in provider_bindings:
             provider = provider_bindings.get(provider_id)(id=provider_id)
-            providers.append({
-                'id': provider_id,
-                'name': provider.name,
-                'config': provider.get_config(),
-            })
+            if provider.name != 'Bitbucket' or features.has('organizations:bitbucket-repos'):
```

Comment:
nit: i think it's slightly nicer to use `provider_id != 'bitbucket'` instead of name